ql/src/test/org/apache/hadoop/hive/ql/io/parquet/read/TestDataWritableReadSupport.java

```diff
@@ -36,7 +37,7 @@ public void testGetProjectedSchema1() throws Exception {
 
     testConversion("structCol", "struct<a:int>", DataWritableReadSupport
       .getProjectedSchema(originalMsg, Arrays.asList("structCol"), Arrays.asList(0),
-        Arrays.asList("structCol.a")).toString());
+        new HashSet<>(Arrays.asList("structCol.a"))).toString());
```

Comment:
Hint: it's most readable is to simply use google Guava after importing ```com.google.common.collect.Sets```:

```
Set<String> StringSet = Sets.newHashSet("structCol.a");
```

The same as other test cases.